scripts/dev-docs.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -ev # exit when error
+
+npm run dev &
+cd docs && bundle install && bundle exec guard
```

Comment:
As for `npm run dev`, having it in postinstall is not sufficient since when you update your repository, package.json or gemfile could be updated and so you would miss some gems.

If you have everything installed, running bundle install just before guard really takes likes 3s on my machine. And you do not restart guard so often (since it's autoreloading thing)